declarations/WebpackOptions.d.ts

```diff
@@ -664,6 +664,12 @@ export interface RuleSetRule {
 	 * Shortcut for resource.exclude
 	 */
 	exclude?: RuleSetConditionOrConditionsAbsolute;
+	/**
+	 * The options for the module generator
+	 */
+	generator?: {
```

Comment:
Maybe we can use `options` instead `generator` for avoiding difference between setup loader and built-in modules?

/cc @sokra

Reply:
These options are only for generator (not to parser or resolver)
`generator` is more semantic that `options`

Reply:
https://github.com/webpack/webpack/blob/master/lib/NormalModuleFactory.js#L406-L407

Reply:
yes, you are right, just idea for naming in configurations